README.md

```diff
@@ -413,8 +500,15 @@ Future.do(function*(){
 This function has an alias `go`, for environments in which `do` is a reserved word.
 
 #### try
-##### `.try :: (() -> !a | b) -> Future a b`
-##### `.attempt :: (() -> !a | b) -> Future a b`
+
+<details><summary><code>try :: (() -> !a | b) -> Future a b</code></summary>
+
+```hs
+try     :: (() -> !a | b) -> Future a b
+attempt :: (() -> !a | b) -> Future a b
```

Comment:
Is the meaning of `!a` explained somewhere? Is the meaning of `a | b` explained somewhere?